packages/office-ui-fabric-react/src/components/Button/ButtonPage.tsx

```diff
@@ -78,6 +81,12 @@ export class ButtonPage extends React.Component<IComponentDemoPageProps, IButton
             <ExampleCard title='Button Swap with Focus State' code={ ButtonSwapExampleCode }>
               <ButtonSwapExample disabled={ this.state.areButtonsDisabled } checked={ this.state.areButtonsChecked } />
             </ExampleCard>
+            <ExampleCard title='SplitButton' code={ ButtonSplitExampleCode }>
+              <ButtonSplitExample disabled={ this.state.areButtonsDisabled } checked={ this.state.areButtonsChecked } />
+            </ExampleCard>
+            <ExampleCard title='' code={ ButtonSplitExampleCode }>
```

Comment:
Probably worth adding some documentation around how to create a split button on this page somewhere.